### DISCUSSION TITLE
Estimating infrastructure cost created by Copilot

### DISCUSSION THREAD

**Discussion by mohamedFaris47** [aws/copilot-cli#2631] [Ideas]
Copilot makes using ECS very easy and I really like it. If there is an option in copilot to estimate the price of the made infrastructure before deployment, it would be awesome. For example ```copilot app --estimate_cost```.  Is there any plans on such feature?

**Comment by huanjani** [comment]
Related: https://github.com/aws/copilot-cli/issues/2509
